collections/nemo_nlp/nemo_nlp/data/data_layers.py

```diff
@@ -131,7 +131,11 @@ def create_ports():
                 0: AxisType(BatchTag),
                 1: AxisType(TimeTag)
             }),
-            "token_mask": NeuralType({
+            "loss_mask": NeuralType({
+                0: AxisType(BatchTag),
+                1: AxisType(TimeTag)
+            }),
+            "subtokens_mask": NeuralType({
```

Comment:
please modify the docstring to explain the different between loss_mask and subtokens_mask